docs/components/SDKs.js

```diff
@@ -51,7 +51,6 @@ export const SDKs = () => {
         <Link to="https://github.com/temporalio/sdk-python">Python</Link>,{" "}
         <Link to="https://github.com/temporalio/sdk-dotnet">.NET</Link>,{" "}
         <Link to="https://github.com/temporalio/sdk-core">Rust</Link>, and{" "}
```

Comment:
```suggestion
        <Link to="https://github.com/temporalio/sdk-dotnet">.NET</Link> and{" "}
        <Link to="https://github.com/temporalio/sdk-core">Rust</Link>.
```
Do we still use this file (SDKs.js) anywhere, @flossypurse? I don't see a current reference to it.

Reply:
Then probably no.

Reply:
Issue moved to [EDU-1068](https://temporalio.atlassian.net/browse/EDU-1068).